target/linux/ramips/image/rt305x.mk

```diff
@@ -389,6 +392,27 @@ define Device/dlink_dir-320-b1
 endef
 TARGET_DEVICES += dlink_dir-320-b1
 
+define Device/dlink_dir-506l
+  SOC := rt5350
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-506L
+  DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-spi-dev kmod-usb-serial \
+		kmod-usb-serial-option kmod-usb-net \
+		kmod-usb-net-cdc-ether comgt-ncm \
+		kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-ledtrig-timer
+  DLINK_ROM_ID := DLK6E2114001
+  DLINK_FAMILY_MEMBER := 0x6E21
+  DLINK_FIRMWARE_SIZE := 0x7E0000
+  KERNEL := $(KERNEL_DTB)
+  IMAGES += factory.bin
+  IMAGE/sysupgrade.bin := mkdlinkfw | pad-rootfs | append-metadata
+  IMAGE/factory.bin := mkdlinkfw | pad-rootfs | mkdlinkfw-factory
+  SUPPORTED_DEVICES += dir-506l
```

Comment:
This line can be dropped.

Reply:
Are you sure?
dlink_dwr-512-b - have this.

Reply:
Yes, because dwr-512-b has been renamed, and this one hasn't been renamed, as it has not been added yet.

Reply:
But all devices have ```SUPPORTED_DEVICES```, or I miss something?

Reply:
Note the "+=" here. The primare entry is created in Device/Default in image/Makefile.

Reply:
```
define Device/dlink_dcs-930
  SOC := rt3050
  IMAGE_SIZE := 3776k
  DEVICE_VENDOR := D-Link
  DEVICE_MODEL := DCS-930
  DEVICE_PACKAGES := kmod-video-core kmod-video-uvc kmod-sound-core \
	kmod-usb-audio kmod-usb-dwc2
  SUPPORTED_DEVICES += dcs-930
  DEFAULT := n
endef
TARGET_DEVICES += dlink_dcs-930
```
can drop ```SUPPORTED_DEVICES``` too?

Reply:
No, because this device has been renamed in https://github.com/openwrt/openwrt/commit/7dd3a234bed4175a7a03f6dd42a4d16d3a80af1e